notification-support/README.md

```diff
@@ -0,0 +1,64 @@
+## stream-registry's notification-support module
+### What is it?
+It set of components which provides an easy way to **listen** events carried out in the `stream-registry` core, 
+specifically _CREATE_, _UPDATE_ and _DELETE_ model entities (like streams, schemas or domains) in order to communicate
+them to available handlers.
+
+### What does it include?
+#### 1. A set of listeners
+In package `com.expediagroup.streamplatform.streamregistry.core.listeners` which includes listeners for `Schema` and 
+`Stream` entities.
+
+#### 2. A _Kafka_ handler for a particular model entity
+It is in package `com.expediagroup.streamplatform.streamregistry.core.handlers` which includes a _Kafka_ event handler 
+for `Schema` entities.
+
+#### 3. Automatic configuration for _Kafka_
+It is an optionally loadable functionality which makes possible set a handler to push Schema change events to a _Kafka_ 
+topic. It will be explained on following section.
+
+### _Kafka_ automatic configuration
+Since this is an optionally loadable functionality, it will be ignored if properties aren't set.
+
+#### `notification.events.kafka.enabled = [true | false] (default: not set)`
```

Comment:
Since the `notification.events.kafka.enabled` is not considered mandatory (app can work if it is not set, that's why definition is `not set`). But you are right, it is confusing. I just set it to `false`

Reply:
Fixed